README.md

```diff
@@ -43,120 +36,74 @@ Check the original technical report
-- [EO Extension Specification][stac-ext-eo] to describe eo data.
 - [Version Extension Specification][stac-ext-ver] to define version tags.
 
 [stac-ext-sci]: https://github.com/radiantearth/stac-spec/tree/v1.0.0-beta.2/extensions/scientific/README.md
-[stac-ext-eo]: https://github.com/radiantearth/stac-spec/tree/v1.0.0-beta.2/extensions/eo/README.md
 [stac-ext-ver]: https://github.com/radiantearth/stac-spec/tree/v1.0.0-beta.2/extensions/version/README.md
 
-### Data Object
 
-| Field Name      | Type                                       | Description                                                                                                                   |
-|-----------------|--------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------|
-| process_level   | [Process Level Enum](#process-level-enum)  | Data processing level that represents the apparent variability of the data.                                                   |
-| data_type       | [Data Type Enum](#data-type-enum)          | Data type (`uint8`, `uint16`, etc.) enum based on numpy base types for data normalization and pre-processing.                 |
-| nodata          | integer \| string                          | Value indicating *nodata*, which could require special data preparation by the network (see [No Data Value](#no-data-value)). |
-| number_of_bands | integer                                    | Number of bands used by the model                                                                                             |
-| useful_bands    | \[[Model Band Object](#model-band-object)] | Describes bands by index in the relevant order for the model input.                                                           |
+### Model Input
 
-#### Process Level Enum
+| Field Name              | Type                            | Description                                                                                                                     |
+|-------------------------|---------------------------------|---------------------------------------------------------------------------------------------------------------------------------|
+| name                    | string                          | Informative name of the input variable. Example "RGB Time Series"                             |
+| bands            | [Band](#bands)                       | Describes the EO data used to train or fine-tune the model.            |
+| input_array           | [Array](#array) | Shape of the input array/tensor ($N \times C \times H \times W$).                                                                     |
+| params           | dict | dictionary with names for the parameters and their values. some models may take multiple input arrays, scalars, other non-tensor inputs.                                                                     |
```

Comment:
Just noticed after the first review. This would need a proper type as well.

Reply:
added this so that parameters are fields in a parameters object, but we don't specify what those fields should be up front. I think nearly any types could be in here so I specified that number, string, boolean, and array are accepted.